docs/Catalog/Packaging.md

```diff
@@ -0,0 +1,188 @@
+<!--pytest-codeblocks:skipfile-->
```

Comment:
maybe skip specific code block?

Reply:
@greptilai
Remove this skipfile, and instead just skip the actual pytest-codeblocks below

Reply:
```suggestion
```